### PR TITLE
Add loading view to notification.html

### PR DESCRIPTION
**Pull request by chikeichan**
This PR adds a initial loading screen to the raw html file - this will allow the loading screen to be painted almost instantaneous, giving user feedback that something is happening instead of a blank screen. This relates to #6590 

Here is some research into the issue (on a 2017 MBP) on current production build 6.5.3:
- Time to First Paint: ~2s
- Time to Content Loaded: ~1.5s

I have created issues #6644, #6645, #6646 , #6647 to capture the rest of the work toward optimizing load time.

instant-load-screen

### CODE REVIEW

**Comment by chikeichan** [comment]
@cjeria - can you comment on my design for the loading screen? 😃

File: app/notification.html [hunk 1]
```diff
@@ -1,16 +1,39 @@
...
+        align-self: center;
+        margin-top: 1rem;
+      }
     </style>
   </head>
   <body class="notification" style="height:600px;">
-    <div id="app-content"></div>
+    <div id="app-content">
+      <img id="loading__logo" src="./images/logo/metamask-fox.svg" />
+      <img id="loading__spinner" src="./images/spinner.gif" />
```
Comment (chikeichan) [inline app/notification.html]: It has to be a gif because scripting time actually blocks css animations too.

**Review by danjm — APPROVED** [review]

**Comment by cjeria** [comment]
What you've put together looks perfectly fine. I'm noticing however there are two different loading indicator versions in your example gif.  Let's use just one for consistency and make it gray #bbc0c5.

This one...
image

And this one that appears for a split second after the signing...
image

I'll leave it up to you to choose which one to use. 

@chikeichan

**Comment by cjeria** [comment]
I'll provide a GIF of the second loading indicator since the loading screen can only use GIF, no code version of a loading indicator.

**Pull request merged**